cli/ballerina-cli-module-pull/src/main/ballerina/src/module_pull/module_pull.bal

```diff
@@ -142,7 +142,7 @@ function pullPackage(http:Client httpEndpoint, string url, string modulePath, st
                     panic createError(resp.message.toString());
                 }
             } else {
-                panic createError("error occurred when pulling the module from remote registry");
+                panic createError("error occurred when pulling the module from remote repository");
```

Comment:
Don't we have the module name the user was trying to pull? So that we can give an error like
```
failed to pull the module 'foo' from the remote repository 'http://central.ballerina.io'
```